docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -64,62 +107,41 @@ propagated to Alluxio.
+It is recommended to collocate Presto with Alluxio so that Presto workers can read data locally. An important option to enable in Presto is `hive.force-local-scheduling`, which forces splits to be
+scheduled on the same node as the Alluxio worker serving the split data. By default, `hive.force-local-scheduling` in Presto is set to `false`, and Presto will not attempt to schedule the work on the same machine as the Alluxio worker node.
 
-It is recommended to collocate Presto with Alluxio so that Presto workers can read data locally. An important option to enable in Presto is `hive.force-local-scheduling`, which forces splits to be 
-scheduled on the same node as the Alluxio worker serving the split data. By default, `hive.force-local-scheduling` in Presto is set to false, and Presto will not attempt to schedule the work on the same machine as the Alluxio worker node.
-
-### Increase `hive.max-split-size`
+#### Increase parallelism
 
 Presto's Hive integration uses the config [`hive.max-split-size`](https://teradata.github.io/presto/docs/141t/connector/hive.html) to control the parallelism of the query. It's recommended to set this size no less than Alluxio's block size to avoid the read contention within the same block.
 
-## Distribute the Alluxio Client Jar
+#### Avoid Presto timeout reading large files
 
-We recommend you to download the tarball from
-Alluxio [download page](http://www.alluxio.org/download).
-Alternatively, advanced users can choose to compile this client jar from the source code
-by following the instructions [here](Building-Alluxio-From-Source.html#compute-framework-support).
-The Alluxio client jar can be found at `{{site.ALLUXIO_CLIENT_JAR_PATH}}`.
+It is recommended to increase `alluxio.user.network.netty.timeout` to a bigger value (e.g. 
+`10min`) to avoid the timeout
+ failure when reading large files from remote worker.
 
-Distribute the Alluxio client jar to all worker nodes in Presto:
-- You must put Alluxio client jar `{{site.ALLUXIO_CLIENT_JAR_PATH}}` into Presto cluster's worker directory
-`$PRESTO_HOME/plugin/hive-hadoop2/`
-(For different versions of Hadoop, put the appropriate folder), And restart the process of coordinator and worker.
+## Examples: Use Presto to Query Tables on Alluxio
 
-## Presto cli examples
+### Create a Hive table on Alluxio
 
-Configure Alluxio as the default filesystem of Hive by following the instructions [here](Running-Hive-with-Alluxio.html#2-use-alluxio-as-the-default-filesystem).
+Here is an example to create an internal table in Hive backed by files in Alluxio.
+You can download a data file (e.g., `ml-100k.zip`) from
+[http://grouplens.org/datasets/movielens/](http://grouplens.org/datasets/movielens/).
+Unzip this file and upload the file `u.user` into `/ml-100k/` on Alluxio:
 
-Create a table in Hive and load a file in local path into Hive:
+```bash
+$ bin/alluxio fs mkdir /ml-100k
+$ bin/alluxio fs copyFromLocal /path/to/ml-100k/u.user alluxio://master_hostname:port/ml-100k
```

Comment:
I don't think we respect scheme or authority passed through the shell

Reply:
removed